theme/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const NAMED_COLORS: Record<string, string> = {
+  // the base color, light or dark
+  DEEP_BLUE: '#0d0c22',
+  // light a
+  GREY: '#7f7f7f',
+  // light b
+  LIGHT_GREY: '#DEDFE2',
+  // light c
+  LIGHTER_GREY: '#F3F3F4',
+  // dark a
+  PALE_GREY: '#ADAEB4',
+  // dark b
+  DARK_GREY: '#3B3B3B',
+  // dark c
+  DARKER_GREY: '#252525',
+  // global
+  BLACK: '#000',
+  WHITE: '#ffffff',
+  // brand
+  LIGHT_BLUE: '#2C72FF',
+  BLUE: '#1D0070',
+  LIGHT_YELLOW: '#FFCD85',
+  YELLOW: '#FFEC1F',
+  LIGHT_RED: '#FFC2E8',
+  RED: '#F15929',
+}
```

Comment:
Is there a need to have this file separate from `styles/constants` ? I'd prefer to have a single source of truth for this.

Reply:
Not sure that all code from styles is needed. Let's moved it into theme folder as needed.

Reply:
I buy that argument.